github/config.go

```diff
@@ -32,9 +32,10 @@ type Owner struct {
 	IsOrganization bool
 }
 
-func RateLimitedHTTPClient(client *http.Client, writeDelay time.Duration, readDelay time.Duration) *http.Client {
+func CustomizedHTTPClient(client *http.Client, writeDelay time.Duration, readDelay time.Duration) *http.Client {
```

Comment:
I've elected to rename this as I think the new name better reflects what it does.

Reply:
I'm good with that!